Runtime/Game/Requests/LeaderboardHistory.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using LootLocker.Requests;
+
+
+namespace LootLocker.Requests
+{
+    public class LootLockerLeaderboardHistoryResponse : LootLockerResponse
+    {
+        public LootLockerLeaderboardArchive[] archives { get; set; }
+    }
+    public class LootLockerLeaderboardArchive
+    {
+        public string last_modified { get; set; }
+        public string content_type { get; set; }
+        public string key { get; set; }
+        public int content_length { get; set; }
+    }
+    public class LootLockerLeaderboardHistoryDetailsResponse : LootLockerResponse
+    {
+        public LootLockerPaginationResponse<string> pagination { get; set; }
+        public LootLockerLeaderboardHistoryDetails[] items { get; set; }
+    }
+    public class LootLockerLeaderboardHistoryDetails
+    {
+        public LootLockerLeaderBoardPlayer player { get; set; }
+        public string metadata { get; set; }
+        public string member_id { get; set; }
+        public int rank { get; set; }
+        public int score { get; set; }
+
```

Comment:
Same question about the empty lines at the end of classes as the details PR. Is there a reason?